metricbeat/mb/module/connector.go

```diff
@@ -61,5 +61,6 @@ func (c *Connector) Connect() (beat.Client, error) {
 		EventMetadata: c.eventMeta,
 		Processor:     c.processors,
 		DynamicFields: c.dynamicFields,
+		TimeSeries:    true,
```

Comment:
As discussed, expose it as a config option and disable it by default by now.

Reply:
yes, unfortunately i didn't find a way to structure the code to take a global configuration option from the yml and apply it to every "Connector" here. Suggestions are welcome.

Reply:
I added this to the TODO in  #10293